docs/how_to/index.md

```diff
@@ -40,6 +40,8 @@ and full working examples, visit the [tutorials section](../tutorials/index.md).
 
 ## Extend and Contribute to MXNet
 
+* [How do I join and contribute to MXNet development discussion?](http://mxnet.io/community/mxnet_channels.html)
```

Comment:
Change to: "How do I join the MXNet development discussion? "

Reason: "Join and contribute to" is redundant